index.html

```diff
@@ -9,6 +9,9 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <h1 class="titulo-libro"><strong>Título del libro</strong> <p class= "titulo-genero"> (género)</p></h1> <br>
+  <h2 class="titulo-secundario"><strong>Autor del libro:</strong> <p class= "secundario-detalle"> Nombre del autor del libro</p> </h2> <br>
```

Comment:
It's not necessary to use `<br>` to separate elements. `<br>` tags could be replaced with margins for example, since the space that you need to create is more of a "content styling" issue than a breaks between the same content (like a piece of news).
You can add a margin to the classes "titulo-genero" and "titulo.secundario" to achieve the same result.

Reply:
I already have margins on my .scss files but still my text looks like this:
![Peek 14-04-2021 11-05](https://user-images.githubusercontent.com/28579312/114724124-a13bfc80-9d11-11eb-9fe0-d018f86a46c8.gif)


 thats why i added the <br> , how can i fix this??

Reply:
i used `div` this time but still doesnt feel right, there is a better way to do this??

Reply:
Wrap the content in a div, let's call it "container", and I'm sure that flex will do the trick